mayastor/src/bdev/nvmx/handle.rs

```diff
@@ -380,19 +370,32 @@ fn io_type_to_err(
     }
 }
 
+/// Initialize memory pool for allocating NVMe controller I/O contexts.
+/// This must be called before the first I/O operations take place.
+pub fn nvme_io_ctx_pool_init(size: u64) {
+    NVME_IOCTX_POOL.get_or_init(|| {
```

Comment:
Can we have a debug!() that shows the pool size etc? This is useful sometimes.

Reply:
We already get the following message from the MemPool::create()
``[2021-06-15T12:02:45.825229905+01:00  INFO mayastor::core::mempool:mempool.rs:51] Memory pool 'bdev_io_ctx' with 65535 elements (24 bytes size) successfully created``
I wasn't sure if there is anything else that can usefully be added without sounding repetitive.